src/main/kotlin/net/ccbluex/liquidbounce/utils/client/FogExtension.kt

```diff
@@ -4,14 +4,6 @@ package net.ccbluex.liquidbounce.utils.client
 import net.minecraft.client.render.Fog
 import net.minecraft.client.render.FogShape
 
-// could be merged into `Color4b` as generics, but not sure if izuna would be fine with that.
-data class FloatColor(val r: Float, val g: Float, val b: Float, val a: Float) {
-//    constructor(color: Color4b) : this(
-//        color.r.toFloat(), color.g.toFloat(),
-//        color.b.toFloat(), color.a.toFloat()
-//    )
-}
-
 fun Fog.copy(
```

Comment:
I don't like the idea of this.
I'd prefer it when you set the states directly (if possible) in the render system.
Or directly construct the Fog object without creating a new Fog object for each value. Instead you could just construct one and default everything that has not changed to the original.